lib/hanami/config/security.rb

```diff
@@ -50,7 +50,7 @@ def content_security_policy(value = nil)
         if value.nil?
           @content_security_policy
         else
-          @content_security_policy = value
+          @content_security_policy = value.split(';').map(&:strip).join('; ')
```

Comment:
@jacobbednarz I was proposing a proper helper but we agreed not to and leave it up to the developer.